src/dfx/assets/new_project_rust_files/README.md

```diff
@@ -0,0 +1,35 @@
+# {project_name}
+
+Welcome to your new {project_name} project and to the internet computer development community. By default, creating a new project adds this README and some template files to your project directory. You can edit these template files to customize your project and to include your own code to speed up the development cycle.
+
+To get started, you might want to explore the project directory structure and the default configuration file. Working with this project in your development environment will not affect any production deployment or identity tokens.
+
+To learn more before you start working with {project_name}, see the following documentation available online:
+
+- [Quick Start](https://sdk.dfinity.org/docs/quickstart/quickstart-intro.html)
+- [SDK Developer Tools](https://sdk.dfinity.org/docs/developers-guide/sdk-guide.html)
+- [Motoko Programming Language Guide](https://sdk.dfinity.org/docs/language-guide/motoko.html)
+- [Motoko Language Quick Reference](https://sdk.dfinity.org/docs/language-guide/language-manual.html)
```

Comment:
I agree with Adam that links to docs.rs for candid, ic_cdk, and ic_cdk_macros would be better here.